Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -260,6 +263,13 @@ public NetworkVisibility visibility
             }
         }
 
+        // interest management: option to force hide from all players
+        // If this object is a player object, it will not be hidden for that player.
+        // => all interest management solutions need this option. might as well
+        //    keep it in NetworkIdentity by default.
+        [Tooltip("Enable to force this object to be hidden from players.")]
+        public bool forceHidden;
```

Comment:
I think this should have a setter that
- Checks if on server, otherwise ignores the change
- Rebuilds observers if actually changed

Reply:
we can consider that in a separate PR.
the goal is to keep feature parity with near zero risk. just to move it into a component.